tests/components/androidtv/patchers.py

```diff
@@ -3,11 +3,11 @@
 from unittest.mock import mock_open, patch
 
 
-class AdbDeviceFake:
-    """A fake of the `adb_shell.adb_device.AdbDevice` class."""
+class AdbDeviceTcpFake:
```

Comment:
The changes in this file reflect the change in the `adb-shell` package from `AdbDevice` to `AdbDeviceTcp`.